test/integration/volumescheduling/volume_binding_test.go

```diff
@@ -783,35 +783,43 @@ func testVolumeProvision(t *testing.T, storageCapacity bool) {
 				t.Fatalf("Failed to create StorageClass %q: %v", sc.Name, err)
 			}
 		}
-		// Create PVs
+
+		t.Log("Creating PVs")
 		for _, pvConfig := range test.pvs {
 			pv := makePV(pvConfig.name, classes[pvConfig.scName].Name, pvConfig.preboundPVC, config.ns, pvConfig.node)
 			if _, err := config.client.CoreV1().PersistentVolumes().Create(context.TODO(), pv, metav1.CreateOptions{}); err != nil {
 				t.Fatalf("Failed to create PersistentVolume %q: %v", pv.Name, err)
 			}
+			if err := waitForPVPhase(config.client, pvConfig.name, v1.VolumeAvailable); err != nil {
```

Comment:
Does this mean the race exists in a real cluster too?

Reply:
It's more like the uncertainty in testing. Our volume binder will try to dynamically provision if no static matching PV is found. But in some scenarios, we expect the volume binder to use the static PV, so we need to wait for the PV to be available. 

I think it's fine in real clusters. In theory, it's possible that some newly created PVs may not be synced to the scheduler or not be in the available state when a PVC is created to request a PV from the storage class. But, normally both the static provisioning and dynamic provisioning are acceptable for users if a storage class supports both of them.